examples/ensemble/plot_ensemble_oob.py

```diff
@@ -0,0 +1,91 @@
+"""
+=========================================================
+OOB Errors for Random Forests and Extra Trees Classifiers
+=========================================================
+
+The ``RandomForestClassifier`` and ``ExtraTreesClasifier`` are trained using
+*bootstrap aggregation*. During training, each new tree is fit from a
+bootstrap sample of the training observations :math:`z_i = (x_i, y_i)`. The
+*out-of-bag* (OOB) error is the average prediction error for each :math:`z_i`
+from trees that do not contain :math:`z_i` in their respective bootstrap
+sample. This allows models to be simultaneously fit and cross-validated [1].
+
+The example below demonstrates how the OOB error can be measured at the
+inclusion of each new tree whilst fitting ``RandomForestClassifier`` and
+``ExtraTreesClassifier`` models. The subsequent plot enables the practitioner
+to approximate the error stabilization point of each model at which training
+can be halted.
+
+.. [1] T. Hastie, R. Tibshirani and J. Friedman, "Elements of Statistical
+       Learning Ed. 2", Springer, 2009.
+
+"""
+import matplotlib.pyplot as plt
+
+from collections import OrderedDict
+from sklearn.datasets import make_classification
+from sklearn.ensemble import RandomForestClassifier, ExtraTreesClassifier
+
+# Author: Kian Ho <hui.kian.ho@gmail.com>
```

Comment:
Btw, feel free to just give yourself as the author, but you can keep us if you like.